sabnzbd/newsunpack.py

```diff
@@ -1034,7 +1036,7 @@ def par2_repair(nzo: NzbObject, setname: str) -> Tuple[bool, bool]:
             joinables, _, _, _ = build_filelists(nzo.download_path, check_rar=False)
 
             # Multipar on Windows, par2cmdline on the other platforms
-            if sabnzbd.WIN32:
+            if cfg.enable_multipar and sabnzbd.WIN32:
```

Comment:
You forgot the ()

Reply:
I think because of that it will always use Multipar instead of turbo.

Reply:
when I tested it worked fine to do one or the other if its enabled. but yes out of the box multipar would be enabled.. figured that would be by design until multipar is removed

Reply:
It needs to be `enable_multipar()`, not just `emable_multipar`.
I really don't know how it could work without.

Reply:
aye, was reading it as something else (mobile). yep I've fixed up